xmtp-inbox-ios/Models/DB.swift

```diff
@@ -68,6 +71,6 @@ class DB {
 	}
 
 	var location: URL {
-		URL.documentsDirectory.appendingPathComponent("db\(mode == .normal ? "" : "-test").sqlite")
+		URL.documentsDirectory.appendingPathComponent("db\(mode == .normal ? "" : "-test-v\(DB.version)").sqlite")
```

Comment:
Do we want the version in the `.normal` mode as well?

Reply:
Oops, great catch.

Reply:
I think we might want to include the XMTP env here too, I'm seeing some crossover between dev and prod on my device.